.ibm/pipelines/jobs/periodic.sh

```diff
@@ -35,12 +35,21 @@ run_runtime_config_change_tests() {
   check_and_test "${RELEASE_NAME}" "${NAME_SPACE_RUNTIME}" "${runtime_url}"
 }
 
-add_sanity_plugins_check() {
-  helm upgrade -i "${RELEASE_NAME}" -n "${NAME_SPACE}" \
-    "${HELM_REPO_NAME}/${HELM_IMAGE_NAME}" --version "${CHART_VERSION}" \
-    -f "${DIR}/value_files/${HELM_CHART_VALUE_FILE_NAME}" \
-    -f "${DIR}/value_files/sanity-check-plugins.yaml" \
-    --set global.clusterRouterBase="${K8S_CLUSTER_ROUTER_BASE}" \
-    --set upstream.backstage.image.repository="${QUAY_REPO}" \
-    --set upstream.backstage.image.tag="${TAG_NAME}"
+run_sanity_plugins_check() {
```

Comment:
Maybe dividing this function into deployment and check_and_test would help to clean it up (at least how `run_runtime_config_change_tests` has it), so we don't have to refactor that much in the future.

Reply:
done